Update README.md

marioquintalcob opened the pull request:
Actualización del readme, revisen para ver si falta, sobra o hay que cambiar algo.

marioquintalcob committed:
Update README.md

Actualización del readme, revisen para ver si falta, sobra o hay que cambiar algo.

YaelRmz commented on README.md:
````diff
@@ -583,15 +817,186 @@ El cual se carga de la siguiente forma en el tema a utilizar en nuestro proyecto
     <style name="Theme.Proyecto01Equipo16" parent="Base.Theme.Proyecto01Equipo16" />
 </resources>
 ```
+Tambien se implemento el uso de Cardviews, utilizando un adaptador y su recyclerview para su correcto funcionamiento:
````
Las CardViews se implementaron en el Home y este es el código de Routines

YaelRmz commented on README.md:
````diff
@@ -583,15 +817,186 @@ El cual se carga de la siguiente forma en el tema a utilizar en nuestro proyecto
 ```
+[device-2023-05-22-120033.webm](https://github.com/marioquintalcob/Proyecto01Equipo16/assets/119343518/e1800041-07ae-4af5-b9f8-a55f9c542dd5)
+
 ### Sesión 8
````
La parte del a session 8 es el buildType que está en el build.gradle a nivel de app

marioquintalcob committed:
Update README.md

Realice las modificaciones que me comentaron, revísenlas para ver si quedo bien.

marioquintalcob committed:
Update README.md

Cambie la imagen de la sesion 6 para que coincida con el codigo.

YaelRmz commented on README.md:
````diff
@@ -583,35 +817,120 @@ El cual se carga de la siguiente forma en el tema a utilizar en nuestro proyecto
     <style name="Theme.Proyecto01Equipo16" parent="Base.Theme.Proyecto01Equipo16" />
 </resources>
 ```
+Las CardViews se implementaron en el Home:
````
Este es el NavBar no el archivo Home 😅

marioquintalcob committed:
Update README.md

Ahora si, ya puse el código de Home.
